### DISCUSSION TITLE
关于 dist_dir 设置问题

### DISCUSSION THREAD

**Discussion by anwen-anyi** [AlistGo/alist#6110] [General]
使用网络链接方式加载资源（应该没写错 :joy_cat:  ）
```json
  "dist_dir": "https://jsd.onmicrosoft.cn/npm/alist-web@3.31.0/dist/",
```

无论是最后的dist是否携带`/` 都会提示如下错误 无法启动
```txt
index.html not exist, you may forget to put dist of frontend to public/dist
```
查看文件里面也有这个 `index.html`文件吖 （说是public文件，按理说填写了这个应该是先调用这个dist_dir里面的吧~）
![image](https://github.com/alist-org/alist/assets/56105412/144bafbc-c24c-40b3-b1ad-d66fdd4930c8)

-----
然后将编译好的`dist`放到本地使用就没上述的错误
```json
  "dist_dir": "data/dist",
```

AList version v3.31.0

**Comment by xhofe** [comment]
不支持网络资源 dist_dir就是本地目录

**Reply by anwen-anyi** [reply]
好的 了解了。

**Comment by GitCourser** [comment]
> 不支持网络资源 dist_dir就是本地目录

请问如何做到前后端分离, 因为路由器空间太小, 想编译个不带前端的程序放路由器上, 前端单独放到其他地方该如何设置?
